### PR TITLE
(maint) Update changelog for 0.6.4 release

### PR DESCRIPTION
**Pull request by McdonaldSeanp**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -6,6 +6,16 @@ This changelog adheres to [Keep a CHANGELOG](http://keepachangelog.com/).
 
 ## [Unreleased]
 
+## [0.6.4] - 2016-07-27
+### Changed
+- Vanagon now uses one defaut output path for all platforms, with an option
```
Comment (stahnma) [inline CHANGELOG.md]: Do we have PRs pending in PA, client-tools, etc to account for this?

Comment (McdonaldSeanp) [inline CHANGELOG.md]: There will be when these projects get bumped to 0.6.4

**Comment by shrug** [comment]
👍

**Comment by jcoconnor** [comment]
👍

File: CHANGELOG.md (hunk 1 above)
Comment (melissa) [inline CHANGELOG.md]: Should this change warrant a bump to 0.7.0? I'm worried about projects that have a less strict vanagon requirement

**Pull request merged**